spec/registry.md

```diff
@@ -255,3 +255,548 @@ which only expects the `accord` option:
 > .input {$object :noun case=nominative}
 > {{You see {$color :adjective article=indefinite accord=$object} {$object}!}}
 >```
+
+# Default Registry
+
+This section describes the functions which each implementation MUST provide to be conformant with
+this specification.
+
+## String Value Selection and Formatting
+
+### Functions
+
+The following functions are provided:
+
+The function `:string` provides string selection and formatting.
+
+### Operands
+
+The _operand_ is any literal or an implementation-defined set of string or
+character-sequence types.
+
+> [!NOTE]
+> This should probably include individual character types, such as `char`.
+
+In addition, implementations MAY perform formatting and selection on 
+`operand` values that do not otherwise have a formatting function registered.
```

Comment:
Is this saying that the `:string` formatter/selector may be used as a "default" by the implementation? If so, it's a little confusing because it's unclear whether to read the sentence that way, or read it to mean that the implementation of `:string` may do arbitrary formatting on its operand.